static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.spec.tsx

```diff
@@ -495,10 +497,16 @@ describe('WidgetBuilder', function () {
         dashboard: testDashboard,
       });
 
-      userEvent.type(
-        await screen.findByRole('textbox', {name: 'Search events'}),
-        'color:blue{enter}'
-      );
+      const search = await screen.findByTestId(/smart-search-input/);
+
+      userEvent.click(search);
+
+      // Use fireEvent for performance reasons as this test suite is slow
+      fireEvent.paste(search, {
```

Comment:
if we use `userEvent.paste` is the test still super slow?

Reply:
No, it shouldn't be, userEvent.type fires keydown, keypress and keyup for each character (that searchbox listens to all 3).

Let me just measure this a couple of times locally to compare some numbers

Reply:
Also interested to know, I wanted to tackle this `userEvent.type` as well but I had trouble with working around the `clipboardData` like you had here, but there's an example of using `userEvent` here:

```
      userEvent.paste(input, 'is:', {
        clipboardData: {getData: () => ''},
      } as unknown as React.ClipboardEvent<HTMLTextAreaElement>);
```

Reply:
yes... this performance issue is with `userEvent.type` is well known, but wouldn't be fine if we use `userEvent.paste` instead of `fireEvent`?